README.md

```diff
@@ -114,6 +115,27 @@ The Data types MUST be in the set defined by the [JSON Schema Specification 7](h
 Throughout the specification `description` fields are noted as supporting Github markdown formatting.
 Where OpenRPC tooling renders rich text it MUST support, at a minimum, markdown syntax as described by [GitHub Flavored Markdown](https://github.github.com/gfm/). Tooling MAY choose to ignore some GitHub Flavored Markdown features to address security concerns.
 
+### Service Discovery Method
+JSON-RPC APIs can support the OpenRPC specification by implementing a service discovery method that will return the OpenRPC schema for the JSON-RPC API. The method MUST be named `rpc.discover`. The `rpc.` prefix is a reserved method prefix for JSON-RPC 2.0 specification system extensions. Below is the OpenRPC specification for the service discovery method:
+
+```json
+{
+  "methods": [
+    {
+      "name": "rpc.discover",
+      "description": "Returns an OpenRPC schema as a description of this service",
+      "result": {
+        "name": "OpenRPC Schema",
+	"schema": {
+	  "$ref": "https://raw.githubusercontent.com/open-rpc/meta-schema/master/schema.json"
+	}
```

Comment:
formatting is a bit wonky here

Reply:
does the spec spec spaces or tabs?

Reply:
see #83

Reply:
![image](https://user-images.githubusercontent.com/1787231/53194873-43944580-35c9-11e9-8789-1461c3b3d211.png)

Reply:
it renders fine in markdown but the formatting of the actual text document is off